iaso/models/org_unit.py

```diff
@@ -509,6 +509,17 @@ def as_small_dict(self):
             res["search_index"] = self.search_index
         return res
 
+    def as_very_small_dict(self):
+        res = {
```

Comment:
Not blocking but I think this dict should be return directly without passing it to res before to return it